apps/web/src/stores/documentVersions.ts

```diff
@@ -1,8 +1,9 @@
 'use client'
 
-import { DocumentVersion, HEAD_COMMIT } from '@latitude-data/core'
+import { HEAD_COMMIT, type DocumentType } from '@latitude-data/core/browser'
```

Comment:
I split core in server `@latitude-data/core` and browser `@latitude-data/core/browser`